docs/src/main/asciidoc/optaplanner.adoc

```diff
@@ -813,6 +813,72 @@ This avoids hard coding a solver time, because the unit test might run on arbitr
 This approach ensures that the test runs long enough to find a feasible solution, even on slow machines.
 But it does not run a millisecond longer than it strictly must, even on fast machines.
 
+Additionally, if you use a `ConstraintProvider` and have `org.optaplanner:optaplanner-test` as a dependency in your pom, you can test per constraint using a
+`ConstraintVerifier`.
+
+Add the `org.optaplanner:optaplanner-test` dependency to your `pom.xml` dependencies section:
```

Comment:
Style, use "active tense" to avoid "you" whenever possible (even though "you" is always better than "we" or "I").

Start with the benefit for the user (test per constraint), before descibing the conditions (add optaplanner-test).

In the context of this text, the use of ConstraintProvider is a given, so no need to repeat it as it makes it more difficult.

```suggestion
To test each constraint in isolation, add `org.optaplanner:optaplanner-test` as a dependency in the `pom.xml`:
```